spec/app/application.rb

```diff
@@ -34,6 +34,16 @@ class Base < Sinatra::Base
         serializer: FooSerializer
       }
     end
+
+    get '/with-each-serializer/' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.